src/sessions/PageManager.ts

```diff
@@ -8,6 +8,7 @@ export type Page = {
     pageId: string;
     interaction: number;
     parentPageId?: string;
+    parentTimeSpentOnPage?: number;
```

Comment:
suggestion: `parentTimeSpentOnPage` -> `timeSpentOnParentPage`

Reply:
agree. Added to the revision